.gitignore

```diff
@@ -1 +1,3 @@
 /vendor/
+.phpunit.result.cache
+composer.lock
```

Comment:
I noticed these weren't in the gitignore before, was it intentional?

Reply:
You can add there files to the gitignore, no problem